### PR TITLE
ui: failure, empty and loading states — six named failure cards, offline banner, connection strip, empty states, skeletons

### LINKED ISSUE #87: ui: failure, empty and loading states — six named failure cards, offline banner, connection strip, empty states, skeletons
## Goal

Every failure, empty and loading state the handoff names has one component in `@agentic/ui`, so pages and the failure-distinction issue (#46) compose them rather than inventing "Something went wrong".

## Requirements

OPS-04, OPS-05, AC-06 — see `docs/requirements.md`.

## Design source

`docs/design/HANDOFF.md` → "Failure distinction", "Edge cases", "Element states" (table row loading), and the bottom half of `Foundations.png` / `artboards/Foundations.dc.html`.

## Owner paths

- `packages/ui/src/kit/states/**` (new), `packages/ui/src/kit/anatomy.ts` and `recipes.ts` (add the `ag-failure` and `ag-banner` scopes), `packages/ui/src/fragment/index.ts`, `packages/ui/src/index.ts`, `packages/ui/__tests__/kit/states/**`, `packages/ui/{README,CHANGELOG}.md`, `docs/architecture.md` §10

## Depends on

#85

## Scope

- **`FailureCard`** (`ag-failure` scope, `kind` axis): the six named states with their own name, icon, signal caption (mono, right-aligned), colour and action, exactly as the table: 1 `client-offline` "This browser is offline" (muted, `Reconnecting…` disabled), 2 `machine` "Machine disconnected" (warning, `Open machine`), 3 `auth` "Sign-in needed on the machine" (warning, `Re-check`), 4 `runtime` "Runtime error" (error, `Retry turn`), 5 `task` "Task failed" (error, `Open task`), 6 `interrupted` "Interrupted" (error, `Resume`). Props: `kind`, `detail` text, `action` (label + `onAction` or `href`), `signal` caption. Never a generic message.
- **`OfflineBanner`** (`ag-banner`): the client-offline banner over the content, muted, "Reconnecting…", rendered from the shell slot.
- **`EventsLostRow`**: the amber "Events lost between seq A and B" line for a session log after reconnect.
- **`ConnectionStrip`** rows on `ag-connection`: this browser (`live` / `reconnecting`), each machine (`online` + session count / `offline` + age); hollow dot for offline.
- **`EmptyState`** variants: workspace ("Create your first agent" + primary button to `/agents`), inbox ("Nothing needs you." in text-muted, heading kept), machines (`platform` row + dashed "Pair a machine" card), chat (composer only; hint as placeholder), generic (title + caption).
- **Skeleton presets**: `TableSkeleton` (3 rows of 14 px bars in `--ag-line`), `CardSkeleton`, `RailSkeleton`, on zero `Skeleton`.
- **`InterruptedItem` / needs-item kind `interrupted`** already exists in the kit; wire the failure vocabulary so the inbox row and the card share one `kind` list exported from `kit/states/kinds.ts`.

## Out of scope

- Mapping live signals (socket status, `Machine.online`, `authStatus`, adapter errors, `Task.status`) to these components and the Resume/Retry actions — that stays in #46, which is edited to depend on this issue.

## Acceptance

- [ ] A scripted test renders all six `FailureCard` kinds and asserts six distinct names, six distinct `data-kind` values, the correct tone per row and the action label; no two kinds share name + action.
- [ ] `EmptyState` variants and skeleton presets render their anatomy; `ConnectionStrip` renders hollow vs solid dots for offline vs online.
- [ ] `zero:validate` 0 errors; `pnpm size` within budget.

## Verify

```sh
pnpm typecheck && pnpm lint && pnpm test packages/ui && pnpm --filter @agentic/ui build && pnpm --filter @agentic/web build && pnpm size
```

## Read first

`docs/design/HANDOFF.md` → Failure distinction, Edge cases; `packages/ui/src/kit/**` (the kit issue's anatomy and recipe pattern); `docs/architecture.md` §10 failure bullet. Then `AGENTS.md` → "Picking up an issue". Part of the tracking issue #11.

### PR DESCRIPTION
**Pull request by andtii**
Closes #87.

Every failure, empty and loading state the handoff names (`docs/design/HANDOFF.md` → "Failure distinction", "Edge cases", the loading row of "Element states") has one component in `@agentic/ui` (`packages/ui/src/kit/states`), so pages and #46 compose them instead of inventing "Something went wrong".

**Scopes** (anatomy + recipe, registered in the fragment, claimed in `tokens.scopes`)
- `ag-failure` — `FailureCard kind=…`: the six named states from one `FAILURES` table (`kit/states/kinds.ts`) — `client-offline` (muted, `Reconnecting…` disabled), `machine` (warning, `Open machine`), `auth` (warning, `Re-check`), `runtime` (error, `Retry turn`), `task` (error, `Open task`), `interrupted` (error, `Resume`) — each with its own name, 16 px icon, mono signal caption, tone and action. Props `kind`, `detail`, `signal`, `action` (`label` / `href` / `onAction` / `disabled` / `loading`), `noAction`. `role="status"`, `aria-label` = the name.
- `ag-banner` — `OfflineBanner`: the "This browser is offline · Reconnecting…" strip (`role="status"`, `aria-live="polite"`); `tone` / `icon` / `message` / `state` open, plus an actions slot, so a machine banner shares the anatomy.
- `ag-empty` — `EmptyState variant=…`: `workspace` ("Create your first agent" card, primary button to `/agents`), `inbox` ("Nothing needs you." line, `compact`), `machines` (dashed "Pair a machine" card, `outline`, to `/pair`), `chat` (the activation hint), `generic` (title + caption + an `actions` slot).

**Compositions**
- `EventsLostRow from to` — the amber mono "Events lost between seq A and B" line, `<li>` for the event log or `<div>`.
- `browserRow` / `machineRow` / `connectionRows` — the two signals → `ConnectionStrip` rows: `live` / `reconnecting…`, `N sessions` / `offline 3h`; hollow dot for the idle states.
- `TableSkeleton cols` / `CardSkeleton` / `RailSkeleton` on zero `Skeleton` — 14 px bars in `--ag-line` (the design-system override), one `aria-busy` `role="status"` region with a hidden "Loading" each.
- `FAILURE_KINDS`, `FAILURES`, `FAILURE_AXES`, `failureSpec`, `NEEDS_KINDS` re-exported from `kit/states/kinds.ts`, so the inbox row and the card share one kind list (`interrupted` is both). `AgFailure` / `AgBanner` / `AgEmpty` aliases satisfy `componentExportName(scope)`.

**Acceptance**
- [x] A scripted test renders all six `FailureCard` kinds and asserts six distinct names, six distinct `data-kind` values, the tone per row and the action label; no two kinds share name + action — `packages/ui/__tests__/kit/states/failure.test.tsx` (`renders all six kinds…`, plus one case per kind for tone, signal, icon and detail; button, link, disabled and `noAction` wiring).
- [x] `EmptyState` variants and skeleton presets render their anatomy; `ConnectionStrip` renders hollow vs solid dots for offline vs online — `packages/ui/__tests__/kit/states/empty-loading.test.tsx`.
- [x] `zero:validate` 0 errors; `pnpm size` within budget — the web build now runs `--strict` and reports `0 errors, 0 warnings` (was 5). After rebasing onto #103 (the fragment restyle), the combined tree measures `@agentic/ui` 62.73 kB and design-system CSS 20.16 kB (brotlied), so the budgets move 62 → 66 kB and 20 → 22 kB in this PR (this PR alone added about 2.2 kB JS and 0.3 kB CSS).

**Decisions**
- **`client-offline` rides the declared `offline` axis value.** The design system already declared `kind: [… offline, machine, auth, runtime, task]` (#84); the card's prop keeps the issue's `client-offline` name and `FAILURES[kind].axis` maps it to `data-kind="offline"` (`data-failure` carries the prop name). No token change, and every declared kind is now wired: `recipes.test.ts` asserts the unwired list is empty and `design-system.test.ts` asserts zero warnings.
- **`--strict` is on** (`apps/web/package.json`, one line outside the owner paths): the plan listed strict validation as the follow-up once warnings reached zero, which this PR does.
- `ag-failure` and `ag-banner` also claim a three-value `tone` (`muted | needs-you | failed`) so a caller can raise a banner or card in a tone without a kind; `ag-empty` claims `compact` and `outline`.
- `kit/vocabulary.ts` gained the three scope claims (it is where `kitScopes` lives; one additive block).
- Skeleton bars rely on zero `Skeleton`'s `defaultLoading` (true) rather than a `model={() => true}` getter, which `controllable` reads as `loaded` in happy-dom.
- `OfflineBanner` is rendered by pages / #46 above the content; the shell gets no new slot in this PR (`packages/ui/src/shell` is not an owner path).

- Review round: an empty skeleton `label` now means "do not announce" (nested presets carry no status semantics), and `machineRow` prints `online` when the session count is unknown instead of "0 sessions".

**Verify**: `pnpm typecheck && pnpm lint && pnpm test && pnpm size` and `pnpm build` green in the worktree (`apps/daemon` conformance `gap` flaked once under the full suite and passes alone, unrelated to this change). Zero-wip issues filed: none.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: packages/ui/src/kit/states/Skeletons.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        <span style={`display: block; inline-size: ${props.width ?? '100%'}; block-size: ${props.height ?? '14px'}`} aria-hidden="true" />
+    </Skeleton.Root>
+), { name: 'SkeletonBar' });
+
+const wrap = (preset: string, label: string, cls: string | undefined, style: string, children: unknown) => (
+    <div data-skeleton={preset} aria-busy="true" role="status" class={cls} style={style}>
+        <span data-visually-hidden="">{label}</span>
+        {children as never}
+    </div>
+);
```
[the remark this answers is not in the document]
Comment (andtii) [reply packages/ui/src/kit/states/Skeletons.tsx]: Fixed: an empty label now means "do not announce" — the nested preset renders no role/aria-busy and no hidden text, and is aria-hidden, so the rail announces once (test asserts one status region).

File: packages/ui/src/kit/states/connection.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    return state === 'live'
+        ? { id: 'browser', name: 'This browser', state: 'live', tone: 'live' }
+        : { id: 'browser', name: 'This browser', state: 'reconnecting…', tone: 'muted', hollow: true };
+}
+
+export function machineRow(machine: MachineConnection): ConnectionRow {
+    if (machine.online) {
+        const n = machine.sessions ?? 0;
+        return { id: machine.id, name: machine.name, state: `${n} ${n === 1 ? 'session' : 'sessions'}`, tone: 'live' };
+    }
```
[the remark this answers is not in the document]
Comment (andtii) [reply packages/ui/src/kit/states/connection.ts:36]: Fixed: `machineRow` prints "N sessions" only when `sessions` is given; an unknown count reads `online` (tests cover 0, 1, N and undefined).
